test/saml20_clj/test.clj

```diff
@@ -174,6 +174,9 @@
   (or (= {:assertion-signed? true :assertion-encrypted? true} (dissoc response-map :response))
       ((some-fn :saml2-assertion? :no-namespace-assertion?) response-map)))
 
+(defn assertion-signed? [response-map]
```

Comment:
Is this any different than `(:assertion-signed? response-map)`?

Reply:
I don't think it is but was trying to match the other predicates here.